### ISSUE TITLE
Scroll bar always present in 1.2.1

### ISSUE THREAD

**Issue by jcbbas** [maykar/custom-header#108]
In version 1.2.1, the scroll bar always appears even though my content does not flow below the visible area. If I revert back to 1.2.0, the issue goes away completely. 
With 1.2.1
![image](https://user-images.githubusercontent.com/42592630/72658736-5bb04880-3983-11ea-9818-d86536dcd2e9.png)
On 1.2.0
![image](https://user-images.githubusercontent.com/42592630/72658740-72ef3600-3983-11ea-88fc-633f30df955a.png)

I could be missing a setting that is causing this, but I have the view set to Kiosk mode so I am unsure what all could be causing it. This seems to happy to both the panel and pi users. 

My Settings 
```
custom_header:
  compact_mode: true
  editor_warnings: false
  exceptions:
    - conditions:
        user: pi
      config:
        compact_mode: false
        default_tab: '4'
        kiosk_mode: true
        show_tabs: '4'
    - conditions:
        user: panel
      config:
        default_tab: '1'
        kiosk_mode: true
        show_tabs: '1'
        disable_sidebar: true
        shadow: false
    - conditions:
        user: br
      config:
        default_tab: '5'
        kiosk_mode: true
        show_tabs: '5'
    - conditions: {}
      config: {}
```

**Comment by jcbbas** [comment]
Hey thanks for taking a look at this. I did want to also mention that I am using card mod, and button-card. Here is my resources config: 
```
resources:
  - type: js
    url: /local/cards/card-modder.js?v=1b
  - type: js
    url: /local/cards/card-tools.js?v=1b
  - type: module
    url: /community_plugin/button-card/button-card.js
  - type: js
    url: /local/cards/simple-clock-card.js
  - type: module
    url: /local/weatherbg-card.js
  - type: module
    url: /local/cards/battery-entity.js?v=0.1
  - type: module
    url: /community_plugin/lovelace-card-mod/card-mod.js
  - type: module
    url: /community_plugin/custom-header/custom-header.js
  - type: js
    url: /community_plugin/calendar-card/calendar-card.js
```
Let me know if there is anything else needed. I tried to look at the F12 menu to see if I could see a difference in the values when using 1.2.0 vs 1.2.1 in regards to scrolling but from what I could tell, they looked to be the same. Most thing set to auto. I can get the scroll bar to go away by adding the following into the body tag: 
 overflow: hidden;

**Comment by maykar** [comment]
This has been added or fixed in the latest version 1.2.2. After updating be sure to check your browser's dev console (usually F12) to make sure you are running the latest version. If it doesn't display 1.2.2 you may either need to clear your cache or reinstall.
